src/data/polynomial/basic.lean

```diff
@@ -158,6 +158,17 @@ def to_finsupp_iso : polynomial R ≃+* add_monoid_algebra R ℕ :=
   map_mul' := by { rintros ⟨⟩ ⟨⟩, simp [mul_to_finsupp], refl },
   map_add' := by { rintros ⟨⟩ ⟨⟩, simp [add_to_finsupp], refl } }
 
+lemma to_finsupp_iso_apply (P : polynomial R) : to_finsupp_iso R P = P.to_finsupp :=
```

Comment:
I think you should just change line 154 above to make this true by refl.

Reply:
Then `simps` can generate both lemmas.

Reply:
What about the suggestion to use `@[simps]`?

Reply:
Can I tell `@[simps]` to generate the lemmas, but not tagging them with `@[simp]? It seems they are actually breaking proof rather the improving them...`